lib/fluent/plugin/output.rb

```diff
@@ -1457,6 +1457,26 @@ def flush_thread_run(state)
           state.mutex.unlock
         end
       end
+
+      def statistics
+        stats = {
+          'emit_records' => @emit_records,
+          # Respect original name
+          # https://github.com/fluent/fluentd/blob/45c7b75ba77763eaf87136864d4942c4e0c5bfcd/lib/fluent/plugin/in_monitor_agent.rb#L284
+          'retry_count' => @num_errors,
+          'emit_count' => @emit_count,
+          'write_count' => @write_count,
+          'rollback_count' => @rollback_count,
+        }
+
+        if @buffer && @buffer.respond_to?(:statistics)
+          (@buffer.statistics && @buffer.statistics['buffer'] || {}).each do |k, v|
```

Comment:
When `@buffer.statistics` reutern false/nil?

Reply:
looks unnecessary. thank you!
https://github.com/fluent/fluentd/pull/2450/commits/7e0941a73b8c0703757dfe6d91a5b12b6fa8b697